### PR TITLE
[New Exercise]: Yacht

### PR DESCRIPTION
**Pull request by glaxxie**

### CODE REVIEW

File: exercises/practice/yacht/Yacht.ps1 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    FOUR_OF_A_KIND  = $null
+    LITTLE_STRAIGHT = $null
+    BIG_STRAIGHT    = $null
+    CHOICE          = $null
+}
+
+Function Get-Score() {
+    <#
+    .SYNOPSIS
+    Implement a function to get the score of a yatch game.
```
Comment (meatball133) [inline exercises/practice/yacht/Yacht.ps1]:
```suggestion
    Implement a function to get the score of a yacht game.
```

File: exercises/practice/yacht/Yacht.ps1 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Given a list of values for five dice and a category, your solution should return the score of the dice for that category.
+    If the dice do not satisfy the requirements of the category your solution should return 0.
+    You can assume that five values will always be presented, and the value of each will be between one and six inclusively.
+    You should not assume that the dice are ordered.
+
+    .PARAMETER Dice
+    An array of 5 integer, each represent a dice value.
+
+    .PARAMETER Category
+    An Enum value represent a category in the game of yatch.
```
Comment (meatball133) [inline exercises/practice/yacht/Yacht.ps1]:
```suggestion
    An Enum value represent a category in the game of yacht.
```

**Review by meatball133 — COMMENTED** [review]
2 tiny nits

File: exercises/practice/yacht/Yacht.ps1 (hunk 2 above)
Comment (glaxxie) [reply exercises/practice/yacht/Yacht.ps1]: Ahh, I thought I catch all the typos. Thanks.

**Review by meatball133 — APPROVED** [review]

**Pull request merged**